src/main/java/com/khomishchak/cryptoportfolio/model/Feedback.java

```diff
@@ -0,0 +1,55 @@
+package com.khomishchak.cryptoportfolio.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.khomishchak.cryptoportfolio.model.enums.FeedbackType;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "feedbacks")
```

Comment:
I recommend also add annotations to prevent lazy loading when it's not needed:

- `@EqualsAndHashCode(exclude = { "user" })`
- `@ToString(exclude = { "user" })`